### ISSUE TITLE
Building Playground requires `patchelf` on linux

### ISSUE THREAD

**Issue by rdeits** [Rory-Finnegan/Playground.jl#43]
Building Playground `v0.0.6_alpha` on linux (Ubuntu 14.04 in my case) fails inside BuildExecutable with

```
ERROR: Using the 'targetdir' option requires the 'patchelf' utility. Please install it.
```

Is it worth trying to build `patchelf` within the Playground.jl build?

**Comment by rdeits** [comment]
Here's an attempt at building patchelf as part of Playground's build: https://github.com/rdeits/Playground.jl/commit/4353b071926d236d6d82b7aea6cda9441b2e9507 

It's currently a bit of a hack, since it uses some un-exported functions from BinDeps, so I don't think it's ready to go yet. But it might still be helpful as a proof of concept.

**Comment by Rory-Finnegan** [comment]
`patchelf` is a pretty common utility for manipulating binaries, so I'd suggest just installing it system wide. I'm also inclined to add it to the build steps in [BuildExecutable.jl](https://github.com/dhoegh/BuildExecutable.jl) since it's the pkg requiring the patchelf dependency.

Honestly, I'd also like to avoid BinDeps for as long as possible.

**Comment by Rory-Finnegan** [comment]
@rdeits I've added the required to BuildExecutable.jl, but at new release hasn't been cut.. so I've decided to remove the BuildExecutable.jl default behaviour on `Pkg.build`. If folks want to build binary release they'll need to do install BuildExecutable and run `julia -e 'ENV["PLAYGROUND_INSTALL"] = true; ENV["PLAYGROUND_BIN_EXEC"] = true; Pkg.build("Playground")'`. I'm going to mark this issue as closed.
